src/library_memfs.js

```diff
@@ -354,20 +354,17 @@ mergeInto(LibraryManager.library, {
         MEMFS.expandFileStorage(stream.node, offset + length);
         stream.node.usedBytes = Math.max(stream.node.usedBytes, offset + length);
       },
-      mmap: function(stream, buffer, offset, length, position, prot, flags) {
-#if ASSERTIONS
-        // The data buffer should be a typed array view
-        assert(!(buffer instanceof ArrayBuffer));
-#endif
+      mmap: function(stream, address, length, position, prot, flags) {
+        assert(address === 0);
```

Comment:
please add a comment here - is this a TODO?

Reply:
No, this is currently an unused argument.  I'm just mimicking the assert that exists in nodefs.   We don't support this address hint (yet).

Added comment.